README.md

```diff
@@ -166,3 +166,26 @@ mvn clean test -D"groups=db2" -D"excludedGroups=postgresql,mysql"
 
 ---
 [https://dialob.io](https://dialob.io)
+
+## Release
+
+### Backend
+
+Merge release version to `main` branch. Github pipeline will execute release and publish artifacts to maven central.
+
+### Frontend
+
+Frontend projects are published using [Changesets](https://github.com/changesets/changesets). 
+
+#### Making a release
+
+1. Use command `pnpm changeset` to select component(s) for a release. 
```

Comment:
This command should be issued from the root of the repository, right?

Reply:
yes